tests/OBSLocal.py

```diff
@@ -1015,7 +1019,7 @@ def commit_files(self, path):
         :param path: path to a directory containing the files that must be added to the package
         """
         for filename in os.listdir(path):
-            with open(os.path.join(path, filename)) as f:
+            with open(os.path.join(path, filename), 'rb') as f:
```

Comment:
opening as binary is needed for gzipped tarball.

Reply:
Why is that a comment here and not in the code?

Reply:
makes sense, will add it.